tutorials/algorithms/09_IQPE.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>A minor issue and it's probably a problem of the <code>plot_histogram</code> function. The probability value <code>1.0</code> is touching the frame. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>